docs/onboarding.md

```diff
@@ -10,7 +10,7 @@ There are a few integration steps you need to follow to enable RPM builds in
 
 2. **Enable the [GitHub Konflux application][gh app]** for your project.
```

Comment:
This should be de-linked too and should instead refer abstractly to the GH app that the user set up for their dedicated instance (see https://github.com/konflux-ci/konflux-ci?tab=readme-ov-file#enable-pipelines-triggering-via-webhooks)